### DISCUSSION TITLE
Schily Tools

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MSYS2-packages#4406] [General]
They are still figuring out how to detect your platform!

https://codeberg.org/schilytools/schilytools/issues/60

**Comment by Biswa96** [comment]
If that project works in cygwin it should work in msys2 also (in most of the cases). There is no need to add extra checks. For example, see the nano/PKGBUILD.

**Reply by ghost** [reply]
Please read the link above again. They said very clear what their problem is.

**Reply by Biswa96** [reply]
The upstream issue does not explain what the actual problem is. If you are trying to compile it provide some steps to reproduce the issue.

**Reply by ghost** [reply]
The whole thread is about the output of `uname`.

**Comment by ghost** [comment]
Nevermind, they are a bunch of Linux/Unix users that don't care about us.
